ui/app/controllers/jobs/job/task-group.js

```diff
@@ -29,11 +33,19 @@ export default class TaskGroupController extends Controller.extend(
     {
       sortDescending: 'desc',
     },
+    {
+      qpStatus: 'status',
+    },
+    {
+      qpClient: 'client',
+    },
   ];
 
   currentPage = 1;
   @readOnly('userSettings.pageSize') pageSize;
 
+  qpStatus = '';
+  qpClient = '';
```

Comment:
As mentioned in the other PR, I think that having `Version` here would be nice.

Reply:
I like the idea of exploring this and appreciate the feedback, but I’m also kinda wanting to just ship the design we had decided on w/ Julie…maybe we can revisit down the road and iterate on it a bit?